iac-extensions/docker/src/main/java/org/sonar/iac/docker/reports/hadolint/HadolintImporter.java

```diff
@@ -43,16 +80,10 @@ protected NewExternalIssue toExternalIssue(JSONObject issueJson) {
     InputFile inputFile = inputFile(path);
 
     String ruleId = reportFormat.getRuleId(issueJson);
-    if (!HadolintRulesDefinition.RULE_LOADER.ruleKeys().contains(ruleId)) {
-      ruleId = "hadolint.fallback";
-    }
 
-    NewExternalIssue externalIssue = context.newExternalIssue()
-      .ruleId(ruleId)
-      .type(HadolintRulesDefinition.RULE_LOADER.ruleType(ruleId))
-      .engineId(HadolintRulesDefinition.LINTER_KEY)
-      .severity(HadolintRulesDefinition.RULE_LOADER.ruleSeverity(ruleId))
-      .remediationEffortMinutes(HadolintRulesDefinition.RULE_LOADER.ruleConstantDebtMinutes(ruleId));
+    NewExternalIssue externalIssue = context.newExternalIssue();
+
+    loadPropertiesIntoIssue(externalIssue, reportFormat, issueJson, ruleId);
```

Comment:
Here the `externalIssue` is created, passed to `loadPropertiesIntoIssue()`, and used later.
I would suggest moving line 84 to `loadPropertiesIntoIssue()` method. Then the method will have fewer arguments and we will see the method result explicitly (as returned value)

Reply:
Maybe then the method name needs to be adjusted

Reply:
I can do that, but than we would have to add the context as another argument.

Reply:
or remove `static` from method declaration